### PR TITLE
[NFC][AMDGPU] Auto-generate check lines for some test cases

### PR DESCRIPTION
**Pull request by shiltian**
- `llvm/test/CodeGen/AMDGPU/andorbitset.ll`
- `llvm/test/CodeGen/AMDGPU/andorxorinvimm.ll`
- `llvm/test/CodeGen/AMDGPU/fabs.f64.ll`
- `llvm/test/CodeGen/AMDGPU/llvm.amdgcn.raw.ptr.buffer.store.ll`
- `llvm/test/CodeGen/AMDGPU/s_mulk_i32.ll`

### CODE REVIEW

**Comment by shiltian** [comment]
* **#112426** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/112426?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @shiltian and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: llvm/test/CodeGen/AMDGPU/fabs.f64.ll [hunk 1]
```diff
@@ -20,77 +36,152 @@ define amdgpu_kernel void @v_fabs_f64(ptr addrspace(1) %out, ptr addrspace(1) %i
...
+; SI-NEXT:    v_mov_b32_e32 v0, s2
+; SI-NEXT:    v_mov_b32_e32 v1, s3
+; SI-NEXT:    buffer_store_dwordx2 v[0:1], off, s[4:7], 0
+; SI-NEXT:    s_endpgm
   %bc= bitcast i64 %in to double
   %fabs = call double @fabs(double %bc)
   store double %fabs, ptr addrspace(1) %out
   ret void
 }
+;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
```
Comment (arsenm) [inline llvm/test/CodeGen/AMDGPU/fabs.f64.ll:186]: Remove the FUNC prefix

File: llvm/test/CodeGen/AMDGPU/andorbitset.ll [hunk 2]
```diff
@@ -1,60 +1,131 @@
...
+; SI-NEXT:    buffer_store_dword v0, off, s[0:3], 0
+; SI-NEXT:    s_endpgm
   %x = or i32 %in, 256
   store i32 %x, ptr addrspace(1) %out
   ret void
 }
 
 @gv = external addrspace(1) global i32
 
-; Make sure there's no verifier error with an undef source.
```
Comment (arsenm) [inline llvm/test/CodeGen/AMDGPU/andorbitset.ll:53]: Lost this comment

Comment (shiltian) [reply llvm/test/CodeGen/AMDGPU/andorbitset.ll:53]: I might not use the script correctly but I thought automatic generation of check lines will remove all comments?

Comment (arsenm) [reply llvm/test/CodeGen/AMDGPU/andorbitset.ll:53]: No, it shouldn't. But it is buggy. More test comments are better, we don't want to lose comments

Comment (jayfoad) [reply llvm/test/CodeGen/AMDGPU/andorbitset.ll:53]: If you put the comment before the `define` line then `update_llc_test_checks` should not remove it.

**Review by arsenm — APPROVED** [review]

**Pull request merged**